### ISSUE TITLE
Radio is broken.

### ISSUE THREAD

**Issue by nextgenthemes** [wp-shortcake/shortcake#543]
``` php
            array(
                'attr'  => 'align',
                'label' => __('Alignment', $this->plugin_slug ),
                'type'  => 'select',
                'options' => array(
                    ''       => __( 'Default', $this->plugin_slug ),
                    'none'   => __( 'None', $this->plugin_slug ),
                    'left'   => __( 'Left', $this->plugin_slug ),
                    'right'  => __( 'Right', $this->plugin_slug ),
                    'center' => __( 'center', $this->plugin_slug ),
                ),
            ),
```

This is part of my register array. Works Perfectly fine, as soon as I replace type with radio. The UI shows the radio boxes as expected but no matter what I select its not added to the short-code.

**Comment by jordif** [comment]
Hi,

I'm experiencing the same issue both with checkboxes and radio buttons. The cause seems to be the following commit:

https://github.com/wp-shortcake/shortcake/commit/9f5890f3d204fadb3f74286704357e0dbb3f3bd5

(Rolling back to the previous array of events fixes this)

Jordi

**Comment by danielbachhuber** [comment]
Thanks for the diagnosis, @jordif. We'll get this fixed in v0.6.2. We'd recommend rolling back to v0.5.0 if this issue impacts you in production.

**Comment by danielbachhuber** [comment]
@jordif Just released v0.6.2 to fix this issue. Thanks again for the report!
